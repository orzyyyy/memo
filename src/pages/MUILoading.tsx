import React from 'react';
import { makeStyles, Theme, createStyles } from '@material-ui/core/styles';
import { CircularProgress } from '@material-ui/core';

const useStyles = makeStyles((theme: Theme) =>
  createStyles({
    root: {
      display: 'flex',
      '& > * + *': {
        marginLeft: theme.spacing(2),
      },
      height: '100vh',
      'justify-content': 'center',
      'align-items': 'center',
    },
  }),
);

const CircularIndeterminate = () => {
  const classes = useStyles();

  return (
    <div className={classes.root}>
      <CircularProgress />
    </div>
  );
};

export default CircularIndeterminate;
