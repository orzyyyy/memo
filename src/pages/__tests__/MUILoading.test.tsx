import React from 'react';
import { mount } from 'enzyme';
import MUILoading from '../MUILoading';

describe('MUILoading', () => {
  it('render correctly', () => {
    const wrapper = mount(<MUILoading />);
    expect(wrapper).toMatchSnapshot();
  });
});
