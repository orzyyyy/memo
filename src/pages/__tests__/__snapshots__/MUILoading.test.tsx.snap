// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MUILoading render correctly 1`] = `
<CircularIndeterminate>
  <div
    className="makeStyles-root-1"
  >
    <WithStyles(ForwardRef(CircularProgress))>
      <ForwardRef(CircularProgress)
        classes={
          Object {
            "circle": "MuiCircularProgress-circle",
            "circleDisableShrink": "MuiCircularProgress-circleDisableShrink",
            "circleIndeterminate": "MuiCircularProgress-circleIndeterminate",
            "circleStatic": "MuiCircularProgress-circleStatic",
            "colorPrimary": "MuiCircularProgress-colorPrimary",
            "colorSecondary": "MuiCircularProgress-colorSecondary",
            "indeterminate": "MuiCircularProgress-indeterminate",
            "root": "MuiCircularProgress-root",
            "static": "MuiCircularProgress-static",
            "svg": "MuiCircularProgress-svg",
          }
        }
      >
        <div
          className="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
          role="progressbar"
          style={
            Object {
              "height": 40,
              "width": 40,
            }
          }
        >
          <svg
            className="MuiCircularProgress-svg"
            viewBox="22 22 44 44"
          >
            <circle
              className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
              cx={44}
              cy={44}
              fill="none"
              r={20.2}
              strokeWidth={3.6}
              style={Object {}}
            />
          </svg>
        </div>
      </ForwardRef(CircularProgress)>
    </WithStyles(ForwardRef(CircularProgress))>
  </div>
</CircularIndeterminate>
`;
